src/Std/Internal/Async/Signal.lean

```diff
@@ -0,0 +1,211 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time
+import Std.Internal.UV.Signal
+import Std.Internal.Async.Select
+
+namespace Std
+namespace Internal
+namespace IO
+namespace Async
+
+/--
+Unix style signals for Unix and Windows.
```

Comment:
Should this mention somewhere that `sigkill` and `sigstop` are intentionally missing because they cannot be caught?